crates/llms/src/embeddings/mod.rs

```diff
@@ -138,11 +136,8 @@ fn encode_embedding(format: &EncodingFormat, array: Vec<f32>) -> EmbeddingVector
 pub trait Embed: Debug + Sync + Send {
     async fn embed(&self, input: EmbeddingInput) -> Result<Vec<Vec<f32>>>;
 
-    fn embed_sync(&self, _input: EmbeddingInput) -> Result<Vec<Vec<f32>>> {
-        let name = any::type_name::<Self>();
-        Err(UnsupportedSyncInvocation {
-            model: name.to_string(),
-        })
+    fn embed_sync(&self, input: EmbeddingInput) -> Result<Vec<Vec<f32>>> {
```

Comment:
I wanted to be able to support async embedding models via the UDF as well. I think that this isn't too different from how it works today (other than I/O herding benefits for an entire batch instead of row-wise blocking)?

When DF 49 lands, we can revert to `UnsupportedSyncInvocation` and have the UDF call the async facade.